glide-core/redis-rs/redis/tests/test_cluster_async.rs

```diff
@@ -2,6 +2,92 @@
 #![cfg(feature = "cluster-async")]
 mod support;
 
+use std::cell::Cell;
+use tokio::sync::Mutex;
+
+use lazy_static::lazy_static;
+
+lazy_static! {
+    static ref CLUSTER_VERSION: Mutex<Cell<usize>> = Mutex::<Cell<usize>>::default();
+}
+
+/// Check if the current cluster version is less than `min_version`.
+/// At first, the func check for the Valkey version and if none exists, then the Redis version is checked.
+async fn engine_version_less_than(min_version: &str) -> bool {
+    let test_version = crate::get_cluster_version().await;
+    let min_version_usize = crate::version_to_usize(min_version).unwrap();
+    if test_version < min_version_usize {
+        println!(
+            "The engine version is {:?}, which is lower than {:?}",
+            test_version, min_version
+        );
+        return true;
+    }
+    return false;
+}
+
+/// static fuction to get the engin version. when version is look like 8.0.0 -> 80000 and 12.0.1 -> 120001
```

Comment:
nit: "to get the engin version" -> "to get the engine version
nit: "when version is look like" -> "when version looks like"
nit: "8.0.0 -> 80000" -> "8.0.0 -> 800000" (you missed a 0)

Reply:
Right, I missed rewriting it when I planned to.
I didnt miss 0